### PR TITLE
Refluffs Pioneer Scarf

### PR DESCRIPTION
**Pull request by Snakebittenn**
No longer a generic brand item.

### CODE REVIEW

**Comment by Snakebittenn** [comment]
!review

File: code/modules/clothing/masks/miscellaneous.dm [hunk 1]
```diff
@@ -189,8 +189,9 @@
 		eye.forceMove(src)
 
 /obj/item/clothing/mask/offworlder
-	name = "pioneer's scarf"
-	desc = "Usually a treasured item, these spider-silk scarves are seen passed down by generations of expeditionaries."
+	name = "scarab scarf"
+	desc = "Usually a treasured item, these spider-silk scarves are seen passed down by generations of Scarabs."
+	desc_fluff = "Scarves like these are passed down from family members as a coming of age ritual, typically given to children to mark them as worthy of setting off on their own."
```
Comment (alsoandanswer) [inline code/modules/clothing/masks/miscellaneous.dm:194]: i suggest you contextualise this to scarabs because it seems disjointed

Comment (Snakebittenn) [reply code/modules/clothing/masks/miscellaneous.dm:194]: Very well

**Review by fernerr — APPROVED** [review]

**Review by furrycactus — APPROVED** [review]

**Review by alsoandanswer — APPROVED** [review]
Thank you

**Pull request merged**